### DISCUSSION TITLE
NeoPixel Matrix can't display freefonts from the adafruit gfx library with TFT_eSPI functional

### DISCUSSION THREAD

**Discussion by flyingpole** [Bodmer/TFT_eSPI#3238] [Q&A - Fonts]
I have an app that uses the Adafruit NeoPixel Matrix library to display various content on a neopixel matrix on an ESP32-S3 Development Module.  The app worked perfectly using free fonts and some modified fonts I created.  I then decided to add a 2.8" 320x240 TFT  ILI9341 LCD display and used the TFT_eSPI library, which works fine.  I developed the code for the TFT display in a separate sketch to test all the display and touch screen features and it worked 100%. I then ported the code to my functional neomatrix program and it compiles and operates internally fine, except the neomatrix displays random pixels whenever using anything other than the built in font in Adafruit Neopixel.  The problem is that I can no longer display anything on the NeoMatrix display using any font other than the built in.  I feel there is some conflict with the Adafruit GFX library and the possibly shared freefonts. somewhere but have exhausted my resources trying to determine what is wrong.  I tried using the RLC fonts and still get random pixels on the Neomatrix vs actual characters.  This occurs when I select any of the freefonts.  

I'm using Arduino IDE.  Tried older V1.8.19  as well as the latest V2.3.2 build.  ESP32 2.0.13 in boards manager. Using TFT_eSPI V2.5.43

Any help in identifying the issue would be appreciated.  User setup and project attached.  Also a link to a video showing my test setup and how the neopixel matrix display does display the built-in font correctly.  

https://www.dropbox.com/scl/fi/th2ttbuwn4atf6twk3zqh/20240311_211127_1.mp4?rlkey=ylgy05wm5gjq4s94doldykrpc&dl=0

[UserSetupFiles.zip](https://github.com/Bodmer/TFT_eSPI/files/14565974/UserSetupFiles.zip)
[TFT_NeoMatrix_Example-240311a.zip](https://github.com/Bodmer/TFT_eSPI/files/14565975/TFT_NeoMatrix_Example-240311a.zip)

[TFT_NeoMatrix_Example-240311a.zip](https://github.com/Bodmer/TFT_eSPI/files/14565977/TFT_NeoMatrix_Example-240311a.zip)
[UserSetupFiles.zip](https://github.com/Bodmer/TFT_eSPI/files/14565978/UserSetupFiles.zip)

**Comment by Bodmer** [comment]
Try commenting out the following line in your setup file thus:
// #define LOAD_GFXFF

I suspect the problem occurs because both Adafruit_GFX and TFT_eSPI are trying to load the same font header files and the compiler only loads one set.
